src/app/api/game-sessions/[gameSessionId]/download/route.ts

```diff
@@ -0,0 +1,103 @@
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+import { stringify } from "csv";
```

Comment:
It should be a differnt import since you changed the package

Reply:
whoops seems like i forgot to push this file